README.rst

```diff
@@ -1,6 +1,11 @@
-.. raw:: html
+.. image:: https://img.shields.io/pypi/pyversions/west?logo=python
+   :target: https://pypi.org/project/west/
 
-   <a href="https://scorecard.dev/viewer/?uri=github.com/zephyrproject-rtos/west"><img src="https://api.securityscorecards.dev/projects/github.com/zephyrproject-rtos/west/badge"></a>
+.. image:: https://api.securityscorecards.dev/projects/github.com/zephyrproject-rtos/west/badge
+   :target: https://scorecard.dev/viewer/?uri=github.com/zephyrproject-rtos/west
+
+.. image:: https://codecov.io/gh/zephyrproject-rtos/west/graph/badge.svg?token=bil7f3uJ7q
```

Comment:
https://docs.codecov.com/docs/status-badges says `token=` is for private repos only?

Reply:
Right, copied this from the configuration, but can remove as it is public.